eventlogger/hapi_server.js

```diff
@@ -8,6 +8,18 @@ module.exports.watch = function  (logger, server, options) {
 
   var ignorePaths = options.ignorePaths;
 
+  function getLogEntry(request, log_type) {
```

Comment:
can we rename to `createLogEntry`? `get` can make it look to the reader like it exists somewhere already